data/commodities.txt

```diff
@@ -522,14 +556,17 @@ trade
 		"turret lathes"
 		"vacuum cleaners"
 		"varnish"
+		"wood"
 	commodity "Electronics" 590 890
 		"accelerometers"
 		"amplifiers"
 		"antennae"
 		"batteries"
+		"calculators"
```

Comment:
Here, also, I'm not sure how well `calculators`, `computers`, `phones`, and `watches` fit this list, since just about everything else is electronic components. Pocket watches are also already listed under luxury goods.

Reply:
They are? I hadn’t realized that.

Reply:
These is already supercomputers though.

Reply:
I’ll leave those there, and move these to Equipment in ae326a6